packages/jest-validate/src/__tests__/validate.test.js

```diff
@@ -266,3 +266,25 @@ test('Repeated types within multiple valid examples are coalesced in error repor
     ),
   ).toThrowErrorMatchingSnapshot();
 });
+
+test('Comments using "//" key are not warned by default', () => {
+  const warn = console.warn;
+  console.warn = jest.fn();
```

Comment:
Could you do `jest.spyOn(console, 'warn').mockImplementation(() => {})` instead of overwriting manually? Then a `console.warn.mockRestore()` at the end will clean up

Reply:
Can you verify the new test in 600944b is what you requested?

I'm still working on ensuring that `"//"` is included in the blacklist even with user supplied blacklists.